addons/skin.estuary/xml/DialogPVRChannelManager.xml

```diff
@@ -160,36 +160,42 @@
 						<width>700</width>
 						<include>SettingsItemCommon</include>
 						<label>$LOCALIZE[19074]</label>
+						<enable>false</enable>
 					</control>
 					<control type="edit" id="8">
 						<description>Channel name</description>
 						<width>700</width>
 						<include>SettingsItemCommon</include>
 						<label>$LOCALIZE[19201]</label>
+						<enable>false</enable>
 					</control>
 					<control type="button" id="9">
 						<description>Channel logo Button</description>
 						<width>700</width>
 						<include>SettingsItemCommon</include>
 						<label>$LOCALIZE[19202]</label>
+						<enable>false</enable>
 					</control>
 					<control type="radiobutton" id="12">
 						<description>EPG activated</description>
 						<width>700</width>
 						<include>SettingsItemCommon</include>
 						<label>$LOCALIZE[19206]</label>
+						<enable>false</enable>
 					</control>
 					<control type="spincontrolex" id="13">
 						<description>EPG source</description>
 						<width>700</width>
 						<include>SettingsItemCommon</include>
 						<label>$LOCALIZE[19200]</label>
+						<enable>false</enable>
 					</control>
 					<control type="radiobutton" id="14">
 						<description>Parental locked</description>
 						<width>700</width>
 						<include>SettingsItemCommon</include>
 						<label>$LOCALIZE[19267]</label>
+						<enable>false</enable>
```

Comment:
Does this mean, every skin needs to be adjusted to work after your code changes in this PR?

Reply:
That's a good point; it's probably better to just disable the controls in code.  I just did the rebase and applied the jenkins auto-format for that comment block; let me make sure doing it with CONTROL_DISABLE works as expected and I'll remove the XML change.  Less for @ronie to look at too!

Reply:
Worked as expected and force-pushed.  Thank you sir.